atomics/include/desul/atomics/Macros.hpp

```diff
@@ -43,7 +45,7 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #define DESUL_FUNCTION
 #endif
 
-#if !defined(__CUDA_ARCH__) && !defined(__HIP_DEVICE_COMPILE__)
+#if !defined(DESUL_HAVE_GPU_LIKE_PROGRESS)
```

Comment:
Is this a testing change? I don't see where it gets set up before this.

Reply:
line 33 or 35 its literally this definition.

Reply:
no, the macro is defined a few lines above, using those two other macros.

Reply:
Got it, just skimmed past it. 🤦